sdk/core/azure-core/tests/async_tests/test_rest_response_backcompat_async.py

```diff
@@ -43,9 +43,7 @@ async def _callback(**kwargs):
 def new_request(port):
     return RestHttpRequest("GET", "http://localhost:{}/streams/basic".format(port))
 
-
-@pytest.fixture
-@pytest.mark.asyncio
+@pytest_asyncio.fixture
 async def get_new_response(new_request):
     async def _callback(transport, **kwargs):
         async with transport() as sender:
```

Comment:
We may end up needing to update the mark for `get_new_response_trio` to `@pytest_trio.trio_fixture`. That's the equivalent of `pytest_asyncio.fixture` that we use for asyncio fixtures.

Reply:
Cross that bridge when we come to it though.

Reply:
I don't think we do? It isn't a plugin that seems be used during our tests, [since it doesn't show up on the plugin line of pytest's output](https://dev.azure.com/azure-sdk/public/_build/results?buildId=2765694&view=logs&j=4fa73ede-de59-50b0-851d-db96582ea108&t=271dca1a-6c62-5d1b-458d-2dd10be9b8db&l=464).

The failures we were seeing in the `azure-core` tests with trio also went away after https://github.com/Azure/azure-sdk-for-python/pull/30159/commits/53e606867b160cb39a76d0268a5eac0345f05075 (I'm unsure why we don't seem to need `pytest-trio`, but I've learned not to look a gift horse in the mouth).